packages/monaco/src/browser/monaco-editor-model.ts

```diff
@@ -75,8 +78,9 @@ export class MonacoEditorModel implements ITextEditorModel, TextEditorDocument {
         this.toDispose.push(this.onDidSaveModelEmitter);
         this.toDispose.push(this.onWillSaveModelEmitter);
         this.toDispose.push(this.onDirtyChangedEmitter);
-        this.resolveModel = resource.readContents(options).then(content => this.initialize(content));
+        this.toDispose.push(this.onDidChangeValidEmitter);
         this.defaultEncoding = options && options.encoding ? options.encoding : undefined;
+        this.resolveModel = this.readContents().then(content => this.initialize(content || ''));
```

Comment:
Should we create a broken model or fire error to prevent opening bogus editor? I tried to preserve it as on master, but only handle a case when we know that a file not found.

Reply:
the thing about ☝️ is that it will log unhandled promise rejection.
would be nice to handle it at least

Reply:
ok, will add console.error